.travis.yml

```diff
@@ -5,9 +5,15 @@ language: python
 matrix:
   include:
     - python: 2.7
-      env: TOXENV=py27
+      env: TOXENV=py27-django111
     - python: 3.6
-      env: TOXENV=py36
+      env: TOXENV=py36-django111
+    - python: 3.6
```

Comment:
Note that we're actually using Python 3.5 for now; we were originally planning on using 3.6, but instead went with the version that comes with our current Ubuntu release.  I suspect we'll later be skipping over 3.6 and going straight to 3.8.

Reply:
Since we already have 3.6 working, I updated the configuration to run the tests on both 3.5 and 3.6.